### DISCUSSION TITLE
Getting test passed always

### DISCUSSION THREAD

**Discussion by fpessacg** [pyuvm/pyuvm#80] [Q&A] [answered]
Hi, I'm getting the test passed answer even when the test fails, I get the corresponding error message but at the end cocotb always said test passed. I tested just adding a + 1 in line 28 where you calculate A+B. I'm just running the example from the repo. 

![Screenshot from 2021-12-23 17-05-23](https://user-images.githubusercontent.com/20261282/147288159-3b182ee5-0d36-4d72-9f94-7b7ad11b21d1.png)

**Comment by raysalemi** [comment, marked as answer]
Good catch!  I used to test this better.

I messed up the `Scoreboard`. I copy and pasted it from somewhere else and lost the last line.  

```
                    self.logger.error(f"FAILED: 0x{A:02x} {op.name} 0x{B:02x} "
                                      f"= 0x{actual_result:04x} "
                                      f"expected 0x{predicted_result:04x}")
# LOST LINE!
                    assert False
```



```

**Comment by raysalemi** [comment]
Fixed
